src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -1480,7 +1480,7 @@ default boolean isFromGuild()
      *         The reaction request was attempted after the Message had been deleted.</li>
      * </ul>
      *
-     * @param  name
+     * @param  unicode
      *         The name of the reaction emoji.
```

Comment:
Should not this get changed to `The unicode of the reaction emoji.` instead?

Reply:
I overlooked the small documentation details it seems, should be fixed now!

Reply:
I _think_ you also overlooked the `@ŧhrows` section of the docs; is the `IllegalArgumentException` still described as `If the provided name is null or empty.`?

Reply:
🤦🏻‍♀️ Thanks, I really gotta change the comment color on my IDE I think